rclcpp/test/rclcpp/test_timers_manager.cpp

```diff
@@ -375,7 +375,7 @@ TEST_F(TestTimersManager, check_one_timer_cancel_doesnt_affect_other_timers)
   // since typical users aren't going to mess around with the timer manager.
   t1 = TimerT::make_shared(
     1ms,
-    [&t1_runs, &t1, cancel_iter]() {
+    [&t1_runs, &t1]() {
```

Comment:
I suppose I knew this, but had to look it up again.

https://en.cppreference.com/w/cpp/language/lambda

A lambda expression can read the value of a variable without capturing it if the variable has const non-volatile integral or enumeration type and has been initialized with a [constant expression](https://en.cppreference.com/w/cpp/language/constant_expression#Constant_expression)

Reply:
I had no idea :).  Today I learned something :).

Reply:
MSVC doesn't seem to like it, though: 

```
**09:25:11 C:\ci\ws\src\ros2\rclcpp\rclcpp\test\rclcpp\test_timers_manager.cpp(380,1): error C3493: 'cancel_iter' cannot be implicitly captured because no default capture mode has been specified [C:\ci\ws\build\rclcpp\test\rclcpp\test_timers_manager.vcxproj]
**
```